.gitlab-ci.yml

```diff
@@ -52,7 +44,8 @@ stages:
         --set image.backend.tag="${CI_COMMIT_SHORT_SHA}"
         --set image.frontend.repository="${CONTAINER_REGISTRY}/${FRONTEND_CONTAINER_REPO}"
         --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}"
-        $KUBE_NAMESPACE ./helm/
+        --values $HELM_VALUES_FILE
+        $KUBE_NAMESPACE parity/substrate-telemetry
```

Comment:
Are you using not specifying the version of the chart on purpose ?

Reply:
`--version` has been added to the `helm upgrade` command and the sed commands which were used to modify `chart.yml` have been removed.